pkg/cli/render.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/acorn-io/aml/pkg/cue"
 	cli "github.com/acorn-io/runtime/pkg/cli/builder"
+	client2 "github.com/acorn-io/runtime/pkg/client"
```

Comment:
nit: maybe `rclient`? also, it doesn't look like `client` is being shadowed by a variable (even if it's not a great name)